release_build_files/readme.md

```diff
@@ -573,6 +573,7 @@ code.
         initialization of Firebase iOS SDK.
     -   Remote Config: Fixed default `Fetch()` timeout being 1000 times too
         high.
+    -   Added retry logic to desktop implementation of storage operations. 
```

Comment:
Make sure to put "Storage (Desktop):" at the beginning.

Suggested phrasing: Storage (Desktop): Added retry logic to PutFile, GetFile, and other operations.

Reply:
Done.